Data/market-hours/market-hours-database.json

```diff
@@ -23329,7 +23329,40 @@
         }
       ],
       "saturday": [],
-      "holidays": [],
+      "holidays": [
```

Comment:
do these exchanges not have data before 2008?

Reply:
Seems we have Oanda Forex data going back until 2002, although for the first 3 years we have only daily data points